### PR TITLE
Update Mozilla.net.xml

### PR DESCRIPTION
**Pull request by TotalCaesar659**

### CODE REVIEW

**Comment by zoracon** [comment]
`Could not resolve host: mozilla.net`

File: src/chrome/content/rules/Mozilla.net.xml [hunk 1]
```diff
@@ -1,45 +1,124 @@
...
+		addons-mozilla-org-san.mktns.services.phx1.mozilla.net
+		addons-redirects.mktns.services.phx1.mozilla.net
+		allizom.mktns.services.phx1.mozilla.net
+		getpersonas.mktns.services.phx1.mozilla.net
+		static-addons-mozilla-net.mktns.services.phx1.mozilla.net
+		nat-solitude.svc.phx1.mozilla.net
+		vmimages.mozilla.net
+		conserver1.untrust.yvr1.mozilla.net
+		lo0.border1.untrust.yvr1.mozilla.net
+		lo0.border2.untrust.yvr1.mozilla.net
```
Comment (J0WI) [inline src/chrome/content/rules/Mozilla.net.xml]: You can ignore all hosts that do not support HTTP/HTTPS

Comment (pipboy96) [reply src/chrome/content/rules/Mozilla.net.xml]: @J0WI I also ignore the ones that only support HTTPS.

**Comment by J0WI** [comment]
I'd still prefer wildcards for Mozilla domains.

**Comment by ivysrono** [comment]
@J0WI Maybe you are right. So close this PR may better?

**Pull request closed without merging**